### PR TITLE
documentation/ZENKO-3377_create-update-delete_replication_gui

### PR DESCRIPTION
**Pull request by wabernat**
Added UI docs for create, update, delete replication workflows

Discovered some hidden metadata search content. Deleted it.

### CODE REVIEW

File: docs/docsource/operation/XDM_GUI/bucket_operations/create_a_replication_workflow.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.. _Create a Replication Workflow:
+
+Create a Replication Workflow
+=============================
+
+Before creating a replication workflow, you must have a source bucket and a
+destination bucket deployed.
```
Comment (terrydervaux) [inline docs/docsource/operation/XDM_GUI/bucket_operations/create_a_replication_workflow.rst]:
```suggestion
destination bucket deployed with **versioning enabled**.
```

File: docs/docsource/operation/XDM_GUI/bucket_operations/metadata_search/http_search_requests.rst [hunk 2]
```diff
@@ -1,12 +0,0 @@
...
-HTTP Search Requests
-====================
-
-Search requests can be also performed by making HTTP requests
-authenticated with the AWS Signature version 4 scheme. See the following
-URLs for more information about AWS V4 authentication.
-
--  http://docs.aws.amazon.com/general/latest/gr/sigv4_signing.html
--  http://docs.aws.amazon.com/general/latest/gr/sigv4-signed-request-examples.html
-
```
Comment (terrydervaux) [inline docs/docsource/operation/XDM_GUI/bucket_operations/metadata_search/http_search_requests.rst]: Why are you deleting this file in this 82adcfc8d5f4db1aa3224d8473fda6344a8f3fdb commit/PR ?

File: docs/docsource/operation/XDM_GUI/bucket_operations/create_a_replication_workflow.rst (hunk 1 above)
Comment (wabernat) [reply docs/docsource/operation/XDM_GUI/bucket_operations/create_a_replication_workflow.rst]: both source and destination must have versioning enabled  (or destination only)?

File: docs/docsource/operation/XDM_GUI/bucket_operations/metadata_search/http_search_requests.rst (hunk 2 above)
Comment (wabernat) [reply docs/docsource/operation/XDM_GUI/bucket_operations/metadata_search/http_search_requests.rst]: It was there.

Comment (wabernat) [reply docs/docsource/operation/XDM_GUI/bucket_operations/metadata_search/http_search_requests.rst]: If you want, I can revert the metadata search mess, but we should definitely review it/put some tickets on the board.

File: docs/docsource/operation/XDM_GUI/bucket_operations/create_a_replication_workflow.rst (hunk 1 above)
Comment (terrydervaux) [reply docs/docsource/operation/XDM_GUI/bucket_operations/create_a_replication_workflow.rst]: Both

File: docs/docsource/operation/XDM_GUI/bucket_operations/metadata_search/http_search_requests.rst (hunk 2 above)
Comment (terrydervaux) [reply docs/docsource/operation/XDM_GUI/bucket_operations/metadata_search/http_search_requests.rst]: > If you want, I can revert the metadata search mess, but we should definitely review it/put some tickets on the board. 

Yes please keep this PR focus on replication. As we discussed this morning, we need to do a big cleanup on the Zenko doc.

**Review by terrydervaux — APPROVED** [review]
LGTM

**Review by terrydervaux — CHANGES_REQUESTED** [review]
please squash your commits before you merge.

**Review by terrydervaux — APPROVED** [review]

**Review by lucieleonard — APPROVED** [review]

**Review by cathydossantospinto — APPROVED** [review]

**Pull request merged**